scripts/sparse-and-shallow-git-fetch.sh

```diff
@@ -10,7 +10,8 @@ if [ "$#" -ne 3 ]; then
   exit 1
 fi
 
-set -ev
+# Exit immediately if command exits with non-zero status.
+set -e
```

Comment:
I'm not sure why it was set to verbose previously. It was echoing the commands onto the screen but I don't think it was intended since there are echo statements in here which explicitly outputs the action that is being performed